sites/next.skeleton.dev/src/content/docs/integrations/popover/react.mdx

```diff
@@ -0,0 +1,193 @@
+---
+layout: '@layouts/LayoutDoc.astro'
+---
+
+export const components = componentSet;
+
+import { Page as ExamplePopover } from '@examples/components/popover/ExamplePopover.tsx';
+import ExamplePopoverRaw from '@examples/components/popover/ExamplePopover.tsx?raw';
+import { Page as ExampleTooltip } from '@examples/components/popover/ExampleTooltip.tsx';
+import ExampleTooltipRaw from '@examples/components/popover/ExampleTooltip.tsx?raw';
+import { Page as ExampleCombobox } from '@examples/components/popover/ExampleCombobox.tsx';
+import ExampleComboboxRaw from '@examples/components/popover/ExampleCombobox.tsx?raw';
+import { Page as ExampleModal } from '@examples/components/popover/ExampleModal.tsx';
+import ExampleModalRaw from '@examples/components/popover/ExampleModal.tsx?raw';
+
+## Floating UI React
+
+<figure class="card-filled-enhanced flex justify-center gap-4 p-8">
+	<a href="https://floating-ui.com/" target="_blank" class="btn preset-filled">
+		Floating UI &rarr;
+	</a>
+	<a href="https://floating-ui.com/docs/react" target="_blank" class="btn preset-filled">
+		Floating UI React &rarr;
+	</a>
+</figure>
+
+<p class="type-scale-5">[Floating UI](https://floating-ui.com/) provides primitives for creating tooltips, popovers, dropdown menus, hover cards, modal dialogs, select menus, comboboxes, and more. Instead of providing pre-built components, this guide will introduce the fundamentals of [Floating UI React](https://floating-ui.com/docs/react). This provides an open-ended solution for creating custom popover components of any form factor, paired with Skeleton's design system and components.</p>
+
+---
+
+## Learn the Basics
+
+If this is your first time using Floating UI, start here:
+
+* [Floating UI Tutorial](https://floating-ui.com/docs/tutorial) - covers the basics of using native Javascript.
+* [Floating UI React Tutorial](https://floating-ui.com/docs/react) - covers the basics of using the React wrapper.
+
+### Imports
+
+Floating UI React exposes all required Floating UI middleware, types, etc.
+
+```ts
+import { useFloating } from '@floating-ui/react';
+```
+
+### Making Elements Float
+
+Make sure you the following [custom class](https://floating-ui.com/docs/tutorial#making-the-tooltip-float) in your global stylesheet. This can then be easily reused for all floating elements. We'll use the following class as the convention for all Skeleton examples below.
+
+```css
+.floating {
+	width: max-content;
+	position: absolute;
+	top: 0;
+	left: 0;
+}
+```
+
+### Z-Index Stacking
+
+Please be aware that Floating UI does not take an opinionated stance on [z-index stacking](https://floating-ui.com/docs/misc#z-index-stacking).
```

Comment:
I wonder if we should add at least a bit more here like

> This means that if a floating element is occluded by another element in the z-axis you have to handle it yourself

Just so that the user has a general idea what this means and is expected of him without having to read documentation on another site.